File: MODUL 2 Praktikum Pemrograman/PRAKTIKUM 2 C/PRAK204_2410817120021_Aulia-Az-Zahra.c
#include <stdio.h>

int main() 
{
    float jarijari, tinggi;
    float phi = 22.0 / 7.0; 

    scanf("%f", &jarijari);
    scanf("%f", &tinggi);
    
    float volume   = phi * jarijari * jarijari * tinggi;
    float luas     = 2 * phi * jarijari * tinggi + 2 * phi * jarijari * jarijari;
    float keliling = 2 * phi * jarijari; 

    printf("Volume   = %.2f\n", volume);      
    printf("Luas     = %.2f\n", luas);          
    printf("Keliling = %.2f\n", keliling);  

    return 0;
}

File: MODUL 2 Praktikum Pemrograman/PRAKTIKUM 2 PY/PRAK204_2410817120021_Aulia-Az-Zahra.py
jarijari = input()

if len(jarijari.split()) == 2:
    jarijari, tinggi = map(float, jarijari.split())
elif len(jarijari.split()) == 1:
    jarijari = float(jarijari)
    tinggi = float(input())

phi = 22.0 / 7.0  

volume   = phi * jarijari**2 * tinggi
luas     = 2 * phi * jarijari * tinggi + 2 * phi * jarijari**2
keliling = 2 * phi * jarijari

print(f"Volume = {volume:.2f}")
print(f"Luas     = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")